opencti-platform/opencti-graphql/src/migrations/1693339530445-fix-old-rename-for-relations.js

```diff
@@ -0,0 +1,61 @@
+import { logApp } from '../config/conf';
+import { elUpdateByQueryForMigration } from '../database/engine';
+import { READ_RELATIONSHIPS_INDICES } from '../database/utils';
+
+const message = '[MIGRATION] Fix old rename for relations';
+
+export const up = async (next) => {
```

Comment:
We already have a migration script for this : 1652114181368-entities_rename

Is it a good idea to create a script?
Or should we manage the two cases of platforms still containing these old types and help them manually ?